### ISSUE TITLE
connect sdk socket broken 

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1701]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.16.1](https://github.com/open-wa/wa-automate-nodejs/compare/3.16.0...3.16.1) (2021-05-23)

- build(deps-dev): bump marked from 2.0.3 to 2.0.4 [`#1700`](https://github.com/open-wa/wa-automate-nodejs/pull/1700)
- build(deps-dev): bump @types/node from 15.3.0 to 15.3.1 [`#1696`](https://github.com/open-wa/wa-automate-nodejs/pull/1696)
- build(deps): bump type-fest from 1.1.2 to 1.1.3 [`#1697`](https://github.com/open-wa/wa-automate-nodejs/pull/1697)
- build(deps): bump swagger-ui-dist from 3.48.0 to 3.49.0 [`#1695`](https://github.com/open-wa/wa-automate-nodejs/pull/1695)
- Release 3.16.1 [`d8b07fa`](https://github.com/open-wa/wa-automate-nodejs/commit/d8b07fa70f50953219296615df6a53221b8e9216)
- 🐛 fix socket connect sdk #1701 [`c1fa9d5`](https://github.com/open-wa/wa-automate-nodejs/commit/c1fa9d5cecf4e7667e953b60571b5ea86f081651)
- 🥚  patches update [`ddd5e33`](https://github.com/open-wa/wa-automate-nodejs/commit/ddd5e3360076bf2e1237dae320384f864375016f)
